manifests/golang.yml

```
@@ -10,6 +10,8 @@ tests/:
         chi: v0.1  # real version not known
   appsec/:
     api_security/:
+      test_apisec_sampling.py:
+        Test_API_Security_sampling: missing_feature
```

If it's okay with you we can enable go here as well - granted my comment below is accurate. I'll make a follow-up PR otherwise

```suggestion
        Test_API_Security_sampling:
          '*': v1.60.0-dev
          net-http: irrelevant (net-http doesn't handle path params)
```